scripts/build/zone-js-builder.js

```diff
@@ -45,10 +48,19 @@ function buildZoneJsPackage(destPath) {
   const buildOutputDir = `${bazelBin}/packages/zone.js/npm_package`;
   const distTargetDir = `${absDestPath}/zone.js`;
 
-  console.info(`# Copy artifacts to ${distTargetDir}`);
+  console.info(`# Copy npm_package artifacts to ${distTargetDir}`);
   rm('-rf', distTargetDir);
   cp('-R', buildOutputDir, distTargetDir);
   chmod('-R', 'u+w', distTargetDir);
 
+  // Copy zone.js.tgz to  `destPath`, so we can test
+  // the the archive generated by `npm_package.pack` rule
+  const distArchiveTargetDir = `${absDestPath}/archive`;
+  console.info(`# Copy npm_package archive file to ${distArchiveTargetDir}`);
+  rm('-rf', distArchiveTargetDir);
+  mkdir('-p', distArchiveTargetDir);
+  cp('-R', `${baseDir}/zone.js-${zonePackageJson.version}.tgz`,
+     `${distArchiveTargetDir}/zone.js.tgz`);
```

Comment:
I am a little confused here. Does the `//packages/zone.js:npm_package.pack` command above generate a `.tgz` file in `baseDir`? What is the purpose of specifying `--output_base` then?

Also, you shouldn't need the `-R` option (since we are copying files, not directories):

```suggestion
  cp(`${baseDir}/zone.js-${zonePackageJson.version}.tgz`, `${distArchiveTargetDir}/zone.js.tgz`);
```

Reply:
Yeah, you are right, thanks.